apaeval_env.yaml

```diff
@@ -5,7 +5,7 @@ channels:
   - defaults
 dependencies:
   - nextflow=21.04.0
-  - snakemake=6.4.0
+  - snakemake=6.10.0
```

Comment:
I think this change should be a separate issue of updating apaeval env snakemake version and separate PR from PAQR execution workflow so that it's easier to find and revert, plus it's not PAQR specific :D

Reply:
Agree, I'll create a new issue. Nevertheless, I need to bump snakemake version in this PR, because otherwise the workflow will not run (snakemake modules aren't implemented in 6.4.0)